interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/urandom -> /var/snap/@{SNAP_NAME}/common/lxc/dev/urandom,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/tty -> /var/snap/@{SNAP_NAME}/common/lxc/dev/tty,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/zero -> /var/snap/@{SNAP_NAME}/common/lxc/dev/zero,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/pts/* -> /var/snap/@{SNAP_NAME}/common/lxc/dev/pts/*,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/pts/0 -> /var/snap/@{SNAP_NAME}/common/lxc/dev/console,
+
+    pivot_root /var/snap/@{SNAP_NAME}/common/lxc/ -> /var/snap/@{SNAP_NAME}/common/lxc/,
+
+    # post pivot_root
+    # @{HOME} doesn't work here
+    mount options=(rw, bind) /home/*/snap/@{SNAP_NAME}/** -> /var/snap/@{SNAP_NAME}/common/lxc/dev/**,
+    mount options=(rw, bind) /home/*/snap/@{SNAP_NAME}/** -> /var/snap/@{SNAP_NAME}/common/devices/*,
+    mount options=(rw, bind) /var/snap/@{SNAP_NAME}/common/devices/* -> /var/snap/@{SNAP_NAME}/common/lxc/dev/*,
+    mount fstype=devpts options=(rw, nosuid, noexec) -> /dev/pts/,
+    mount options=(rw, bind) /dev/{,pts/}ptmx -> /dev/{,pts/}ptmx,
+
+    # apparmor="DENIED" operation="mount" info="failed mntpnt match" error=-13 profile="snap.anbox.container-manager//lxc" name="/" pid=4539 comm="anbox" flags="rw, rslave"
+    umount /,
+    umount /var/snap/@{SNAP_NAME}/common/lxc/dev/console,
+    # done post pivot_root
+
+    # starting processes in the container
+    /anbox-init.sh ixr,
+    @{PROC}/1/attr/current r,
+
+    # end processes in the container
+
+    owner @{PROC}/@{pid}/net/unix r,
+    unix (bind,listen) type=stream addr="@/var/snap/anbox/common/containers/default/command",
+    /sys/fs/cgroup/*/lxc/{,**} rw,   # this should be anbox specific
+    @{PROC}/@{pid}/uid_map rw,
+    @{PROC}/@{pid}/gid_map rw,
+    /sys/fs/cgroup/cpuset/cpuset.cpus r,
+    /sys/fs/cgroup/cpuset/cpuset.mems r,
+
+    # None of these work with this denial:
+    # operation="ptrace" profile="snap.anbox.container-manager" pid=27006 comm="anbox" requested_mask="trace" denied_mask="trace" peer="snap.anbox.container-manager"
+    ptrace peer=@{profile_name},
+    #ptrace peer=snap.@{SNAP_NAME}.*,
+    #ptrace,
+
+    capability chown,
+    capability mknod,
+    capability fowner,
+    capability fsetid,
+    capability dac_override,
+    capability dac_read_search,
+    capability setuid,
+    capability setgid,
+    capability sys_admin,
+
+    # We need to allow access to the fuse device here as we're in a child
+    # profile and fuse-support wouldn't help at this point.
+    /dev/fuse rw,
+
+    # container start
+    owner @{PROC}/[0-9]*/fd/ r,
+    owner @{PROC}/[0-9]*/stat r,
+    owner @{PROC}/[0-9]*/ns/ r,
+    /dev/ptmx rw,
+    /dev/pts/[0-9]* rw,
+    capability net_admin,
+    /sys/devices/** r,
+    capability sys_ptrace,
+    / r,
+    capability sys_module, # why?
```

Comment:
Great question-- with binder and ashmem already loaded by snapd, this shouldn't be needed. Please investigate.